source/includes/fact-ssl-ssh-connection-warning.rst

```diff
@@ -0,0 +1,8 @@
+.. important::
+
+   Although |compass| supports connecting to a MongoDB instance through
+   an SSH tunnel via TLS/SSL, this configuration may lead to
+   unexpected behavior when connecting to a :manual:`secondary
+   </core/replica-set-secondary/>` member due to the possibility of the
+   :manual:`primary </core/replica-set-primary/>` changing during an
+   election.
```

Comment:
is the issue that if you are intentionally connecting to a secondary that secondary may become primary? This isn't completely clear to me.

Reply:
Yes, that's the issue as I'm understanding it. CC @imlucas on this in case there's something else here I'm missing.